spec/integration/opentracing_spec.rb

```diff
@@ -0,0 +1,249 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'elastic_apm/opentracing'
+
+RSpec.describe 'OpenTracing bridge', :intercept do
+  let(:tracer) { ::OpenTracing.global_tracer }
+
+  before :context do
+    ::OpenTracing.global_tracer = ElasticAPM::OpenTracing::Tracer.new
+  end
+
+  context 'without an agent' do
+    it 'is a noop' do
+      thing = double(ran: nil)
+
+      tracer.start_active_span('namest') do |scope|
+        expect(scope).to be_a ElasticAPM::OpenTracing::Scope
+
+        tracer.start_active_span('nested') do |nested_scope|
+          expect(nested_scope.span).to be ::OpenTracing::Span::NOOP_INSTANCE
+
+          thing.ran('…')
+        end
+      end
+
+      expect(thing).to have_received(:ran).with('…')
+    end
+  end
+
+  context 'with an APM Agent' do
+    before { ElasticAPM.start }
+    after { ElasticAPM.stop }
+
+    describe '#start_span' do
+      context 'as root' do
+        subject! { ::OpenTracing.start_span('namest') }
+
+        it { should be_an ElasticAPM::OpenTracing::Span }
+        its(:elastic_span) { should be_an ElasticAPM::Transaction }
+        its(:context) { should be_an ElasticAPM::OpenTracing::SpanContext }
+
+        it 'is not active' do
+          expect(::OpenTracing.active_span).to be nil
+        end
+      end
+
+      context 'as a child' do
+        let(:parent) { ::OpenTracing.start_span('parent') }
+        subject! { ::OpenTracing.start_span('namest', child_of: parent) }
+
+        its(:context) { should be parent.context }
```

Comment:
Can you also check this is an `elastic_span`, similar to how you check the first one is an `elastic_transaction`, https://github.com/elastic/apm-agent-ruby/pull/273/files#diff-6b82f284425585e507cbf2452d1c1f8eR40